Mathlib/Algebra/CharP/MixedCharZero.lean

```diff
@@ -213,15 +213,17 @@ noncomputable def algebraRat (h : ∀ I : Ideal R, I ≠ ⊤ → CharZero (R ⧸
     map_one' := by simp
     map_mul' := by
       intro a b
-      simp [field_simps]
+      simp only [divp_assoc', divp_mul_eq_mul_divp, divp_divp_eq_divp_mul, divp_eq_iff_mul_eq,
```

Comment:
This was a non-terminal simp anyway, so I used the occasion to also squeeze them.